Zend/zend_alloc.c

```diff
@@ -452,7 +452,18 @@ static void *zend_mm_mmap_fixed(void *addr, size_t size)
 static void *zend_mm_mmap(size_t size)
 {
 #ifdef _WIN32
-	void *ptr = VirtualAlloc(NULL, size, MEM_COMMIT | MEM_RESERVE, PAGE_READWRITE);
+	void *ptr;
+#ifdef _WIN64
+	// For now enabling only for 64 bits, seems more costly than benefitial otherwise
+	// ZEND_MM_CHUNK_SIZE being the minimum large page size too.
```

Comment:
Are we sure about this? Don't we need to check [`GetLargePageMinimum()`](https://docs.microsoft.com/en-us/windows/win32/api/memoryapi/nf-memoryapi-getlargepageminimum)?

Reply:
I did in a small app, x86 64 supports 2MB and 1GB and GetLargePageMinimum returns the former.

Reply:
Yeah, but doesn't this depend on the processor? What if a processor doesn't support large pages at all, or requires a larger size than 2MiB (`ZEND_MM_CHUNK_SIZE`)?

Reply:
Fair point I ll add a check.